dockerfiles/base/scripts/base/commands/cmd_start.sh

```diff
@@ -165,13 +165,13 @@ cmd_start_check_postflight() {
 cmd_stop() {
   debug $FUNCNAME
   FORCE_STOP=false
-  if [[ "$@" == *"--force"* ]]; then
+  if [[ "$@" == *"--skip:graceful"* ]]; then
   	FORCE_STOP=true
   fi
 
   if [[ ${FORCE_STOP} = "false" ]]; then
-    info "Waiting for graceful stop of services..."
-    cmd_action "graceful-stop" "$@"
+    info "stop" "Stopping workspaces..."
+    cmd_action "graceful-stop" "$@" >> "${LOGS}" 2>&1 || true
```

Comment:
I was thinking that if there is a graceful stop fail, then we just turn it into a non-graceful stop :).  But I see your point.  You think it should be || false and then put it into an if statement?